src/components/Hero.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import styled from 'styled-components';
+import heatmap from '../assets/table.png';
+
+
+const Main = styled.main`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+`;
+
+const Title = styled.h1`
+  color: ${({ theme }) => theme.color.secondary};
+  font-family: ${({ theme }) => theme.font.secondary};
+  font-size: 38px;
+  margin: 27px 0 9px 0;
+  font-weight: normal;
+`;
+
+const SubTitle = styled.h2`
+  font-size: 16px;
+  font-weight: normal;
+`;
+
+const ImageTitle = styled.h3`
+  font-size: 16px;
+  font-weight: 500;
+  margin: 15px -1px 36px 0;
+  letter-spacing: 0.25px;
+`;
+
+const CTAButton = styled.button`
+   color: ${({ theme }) => theme.button.color};
+   background-color: ${({ theme }) => theme.button.background};
+   font-size: 14px;
+   text-transform: uppercase;
+   border: none;
+   border-radius: 4px;
+   margin: 32px 0;
+   padding: 0 15px 0 16px;
```

Comment:
It's interesting that the padding on the left and on the right is different... But I see that it doesn't fit with equal padding on both sides

Reply:
I thought about this a bit more and I realized that I emphasized pixel-perfectness here and with other students too much. I want you to learn how to create pixel-perfect designs, yes. And you clearly know how. 

But in a real-world project you wouldn't set left and right padding on an element like this button. From the designs it's clear that the text should be centered. 

So in this case it looks like a mistake made by the designer. Maybe they changed the text inside the button and forgot to center align it afterward.

So the lesson to learn here for the real world: don't trust the designs in every case and keep the goal in mind. The goal here is to center the text so select either 15px or 16px as padding on both sides.

I hope this makes sense to you :smile:

Reply:
Makes total sense!